deployments/src/addresses.json

```diff
@@ -1,41 +1,41 @@
 {
   "Assessment": "0xcafeaa5f9c401b7295890f309168Bbb8173690A3",
+  "AssessmentViewer": "0xcafeaC01F6BA70567f5EA7dD4e114f8E5FDa5bf6",
   "Chainlink-DAI-ETH": "0x773616E4d11A78F511299002da57A0a94577F1f4",
   "Chainlink-ENZYME-VAULT": "0xCc72039A141c6e34a779eF93AEF5eB4C82A893c7",
   "Chainlink-ETH-USD": "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419",
   "Chainlink-RETH-ETH": "0x536218f9E9Eb48863970252233c8F271f554C2d0",
   "Chainlink-STETH-ETH": "0x86392dC19c0b719886221c78AB11eb8Cf5c52812",
   "Cover": "0xcafeac0fF5dA0A2777d915531bfA6B29d282Ee62",
-  "CoverMigrator": "0xcafeac41b010299A9bec5308CCe6aFC2c4DF8D39",
```

Comment:
is it correct to drop the deleted contracts in the addresses.json or do we still want to keep track of them?

Reply:
it's fine to drop them